### PR TITLE
Dodeca mixer using the AUX channels

### PR DESCRIPTION
**Pull request by imcnanie**
This is an experimental Coaxial Dodecarotor Mixer for the Pixhawk (Tested with Pixhawk 2). I've run more than 20 battery packs on this code, running auto missions, position hold, and stabilized mode. Haven't noticed any issues yet, but I would love to get feedback on my approach.

The strategy is separately mixing the top and bottom halves of the copter. The bottom six rotors run on PX4FMU (AUX Channels) and the top six rotors run on PX4IO (MAIN Channels).

Stabilized Mode:
[IDodeca Copter in Stabilized Mode](https://www.youtube.com/watch?v=D82upRtsy7g)

Running an Auto Mission:
[IDodeca Copter Running and Auto Mission](https://www.youtube.com/watch?v=en-JU5YkVwo)

Flight Logs: http://review.px4.io/plot_app?log=87608c75-8c0d-43e5-b96f-e851421c01a9

Previous topics about this mixer: http://discuss.px4.io/t/more-than-8-rotors-using-the-aux-channels/3479 https://github.com/PX4/Firmware/issues/7116

### CODE REVIEW

**Comment by dagar** [comment]
One trivial formatting difference the build system doesn't like.
https://travis-ci.org/PX4/Firmware/jobs/250453454

File: ROMFS/px4fmu_common/init.d/24001_dodeca_cox [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	param set RTL_DESCEND_ALT 10.0
+	param set PWM_MIN 1075
+	param set PWM_MAX 1950
+	param set PWM_RATE 400
+
+	param set RTL_LAND_DELAY 0
+fi
+
+set PWM_AUX_RATE 400
+# Note: May Have to set these parameters manually. They don't appear to save.
```
Comment (dagar) [inline ROMFS/px4fmu_common/init.d/24001_dodeca_cox]: These aren't parameters, they're variables within the init scripts.

**Comment by dagar** [comment]
Glad to see it working! Could you get rid of the merge commit in your branch? We try to keep the history simple in master.
Initially looks good. Once CI is fully passing I'll take a closer look.

**Comment by dagar** [comment]
Would it make more sense to name them top/bottom? The filename extension already includes main/aux.

**Comment by imcnanie** [comment]
@dagar Makes sense, I'll rename it. Currently working on removing the merge commit.

File: ROMFS/px4fmu_common/init.d/24001_dodeca_cox (hunk 1 above)
Comment (imcnanie) [inline ROMFS/px4fmu_common/init.d/24001_dodeca_cox]: Interesting, should I be using param set instead?

**Comment by imcnanie** [comment]
@dagar How would I go about removing the merge commit?

**Comment by LorenzMeier** [comment]
I've squashed it into one commit, so the merge commit is also gone.

**Pull request merged**